### ISSUE TITLE
DataChart with detail should announce Detail content with screen reader

### ISSUE THREAD

**Issue by mcbouslog** [grommet/grommet#7309]
Using a DataChart, with `detail`, when I use arrow keys or mouseover a series value the Detail component shows the serie label or property and serie value, and maybe it should also announce that information with a screen reader?

### Is this something you or your team is interested in contributing a pull request towards?
I have started a related pull request, will open as draft shortly, and will link to this issue.

### Expected Behavior

The content shown in the DataChart Detail component (on arrow or mouseover series data) is announced with a screen reader.

### Actual Behavior

The content shown in the DataChart Detail component (on arrow or mouseover series data) is not announced with a screen reader.

### URL, screen shot, or Codepen exhibiting the issue
Grommet DataChart Detail story - https://storybook.grommet.io/?path=/story/visualizations-datachart-detail--detail

### Steps to Reproduce

1. from https://storybook.grommet.io/?path=/story/visualizations-datachart-detail--detail
2. turn on screen reader
3. navigate to DataChart
4. use arrow keys to focus different bars within DataChart
5. note Detail component shows bar content
6. note bar content is not announced with screen reader

### Your Environment

- Grommet version: 2.40.1
- Browser Name and version: Chrome v128
- Operating System and version (desktop or mobile): desktop

**Comment by jcfilben** [comment]
Resolved by #7310
